README.md

```diff
@@ -8,15 +8,15 @@ they are a distraction for users.
 
```

Comment:
> humans using the API. While these fields are necessary for Kubernetes, the machine, they are a distraction for users.

Change to

humans using the API. While these fields are necessary for Kubernetes, they are a distraction for users.